rcloud.packages/rcloud.python/R/main.R

```diff
@@ -1,59 +1,112 @@
-.start.python <- function(rcloud.session)
+#' 
+#' Configuration
+#' 
+#' Python version and library path are defined by the following properties:
+#'  * rcloud.python.path - path to Python installation that should be used
+#'  * python.extra.libs - additional Python lib directories (e.g. where Jupyter Python modules got installed)
+#'  
```

Comment:
Please mention these are `rcloud.conf` settings. (Not that we have many other kinds of properties.)